playbook/telegraf.yml

```
@@ -1,4 +1,4 @@
-- hosts: client
+- hosts: test
```

il faudrait laisser `all` ici, lorsqu'on lancera ce play-boy ce sera sur un inventaire qui n'aura probablement de groupe `test`

```suggestion
- hosts: all
```

---

Est ce qu'il ne faudrait pas, ici, mettre plutot "tout sauf main" (car les clés sont déjà installés depuis main-infra) ?

---

```suggestion
- hosts: all:!main
```